Content.Shared/Chat/TypingIndicator/TypingIndicatorComponent.cs

```diff
@@ -14,7 +15,13 @@ public sealed partial class TypingIndicatorComponent : Component
     /// <summary>
     ///     Prototype id that store all visual info about typing indicator.
     /// </summary>
-    [ViewVariables(VVAccess.ReadWrite)]
-    [DataField("proto", customTypeSerializer: typeof(PrototypeIdSerializer<TypingIndicatorPrototype>))]
-    public string Prototype = SharedTypingIndicatorSystem.InitialIndicatorId;
+    [DataField]
+    public ProtoId<TypingIndicatorPrototype> DefaultTypingIndicator = "default";
+
```

Comment:
Just put this on the list instead of having a dummy.

Reply:
The reason I didn't do that is because I didn't think you could initialize a list in the YML file. How would I do that? Example:
```
  - type: TypingIndicator
    proto: robot
    defaultTypingIndicator: robo
```
I don't know how to make the defaultTypingIndicator add to a list. This is also the case for the typingindicator.yml file.

Reply:
I think it's just 
`defaultTypingIndicator: [robo]`

Reply:
It is! Thank you 👍